### DISCUSSION TITLE
When will this feature be made? [DatePicker] [RangePicker]

### DISCUSSION THREAD

**Discussion by dkosinskyi** [ant-design-blazor/ant-design-blazor#3211] [Q&A]
![image](https://user-images.githubusercontent.com/112430454/229093106-47b60e79-1f1a-4dc6-9c62-4c5459c803ba.png)
i`m really need this feature

**Comment by ElderJames** [comment]
@Alexbits Would you be interested in trying it?

**Reply by Alexbits** [reply]
> @Alexbits Would you be interested in trying it?

@ElderJames I will try when I have free time.
